letsencrypt/client/tests/crypto_util_test.py

```diff
@@ -63,9 +62,9 @@ class MakeKeyTest(unittest.TestCase):  # pylint: disable=too-few-public-methods
 
     def test_it(self):  # pylint: disable=no-self-use
         from letsencrypt.client.crypto_util import make_key
+        # This individual test was taking over 6 seconds...
```

Comment:
I do understand the comment when reviewing the PR diff, but will not understand it later when reading the source code in my editor, if you know what I mean... You can just say `# Don't try to test making bigger keys as this will slow down tests severely`